pyproject.toml

```diff
@@ -120,7 +120,6 @@ optional = [
     # optuna/visualization/param_importances.py.
 ]
 test = [
-    "codecov",
```

Comment:
Since this repository has already used `codecov/codecov-action@v3` to upload the coverage reports to Codecove, we do not need further migration.

> Users that would like to continue using Codecov must migrate over to use the new [uploader](https://docs.codecov.com/docs/codecov-uploader) or [GitHub Action](https://github.com/codecov/codecov-action), if applicable.

c.f., https://about.codecov.io/blog/message-regarding-the-pypi-package/